actionpack/lib/abstract_controller/translation.rb

```diff
@@ -16,7 +26,22 @@ def translate(key, options = {})
         options[:default] = defaults
         key = "#{path}.#{action_name}#{key}"
       end
-      I18n.translate(key, options)
+
+      i18n_raise = false
+
+      if html_safe_translation_key?(key)
+        html_safe_options = options.dup
+        options.except(*I18n::RESERVED_KEYS).each do |name, value|
+          unless name == :count && value.is_a?(Numeric)
+            html_safe_options[name] = ERB::Util.html_escape(value.to_s)
+          end
+        end
+        translation = I18n.translate(key, html_safe_options.merge(raise: i18n_raise))
```

Comment:
Why do we want to set raise to false here? I also don't think there's any gain in setting `i18n_raise` to a var over passing `false` directly since the user cannot change the value.